WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/ui/wizards/projectregistration/ProjectCreatedEndingPage.java

```diff
@@ -58,11 +79,68 @@ protected void makeRegistration() {
 		}
 
 		successfulRegistration = true;
-		((ProjectRegistrationWizard) getWizard()).projectId = id;
+
+		((RegistrationWizard) getWizard()).setProjectId(id);
```

Comment:
Actually, it was unsafe before, with accessing field in project wizard. Now with the base class, setProjectId(id) method from this class sets the field in the same class. Cast operation is safe, since the wizard can only be RegistrationWizard.